Godeps/Godeps.json

```diff
@@ -128,13 +128,18 @@
 		},
```

Comment:
All the Godeps changes should be in their own commit. This will help us track down potential issues caused by this change.

The commit which updates these libraries should be the first one. This PR is quite big and we should split it into multiple commits.